src/pages/studyView/table/FixedHeaderTable.tsx

```diff
@@ -38,14 +34,12 @@ const RVSDTtoStrType = {
 
 @observer
 export default class FixedHeaderTable<T> extends React.Component<IFixedHeaderTableProps<T>, {}> {
-    private _store: LazyMobXTableStore<T>;
+    @observable private _store: LazyMobXTableStore<T>;
```

Comment:
Make the whole store observable?

Reply:
@zhx828 store is not `observable` anymore, instead we are calling `store.updateProps` function which is an `action` itself.